specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -792,20 +859,20 @@ paths:
       parameters:
         - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/master/api-specificatie/common.yaml#/components/parameters/fields"
         - in: query
-          name: burgerservicenummer
+          name: ingeschrevenpersoon__burgerservicenummer
           description: "Het burgerservicenummer van de ingeschreven persoon."
           required: false
           schema:
             type: string
             maxLength: 9
         - in: query
-          name: kadasterpersoonidentificatie
+          name: persoon__kadasterpersoonidentificatie
```

Comment:
zie opmerking bij ingeschrevenpersoon_burgerservicenummer. Moet denk ik zijn beslaglegger__identificatie.

Risico is dat iemand hier dan een burgerservicenummer in gaat stoppen, zou de API dan eigenlijk ook moeten ondersteunen.

Reply:
Zie eerdere reactie